doc/modules/decomposition.rst

```diff
@@ -80,6 +81,9 @@ in order update ``explained_variance_ratio_`` incrementally. This is why
 memory usage depends on the number of samples per batch, rather than the
 number of samples to be processed in the dataset.
 
+As in :class:`PCA`, IncrementalPCA centers but does not scale it the input data
```

Comment:
I've just noticed, here it would be useful to use :class:`IncrementalPCA`, to have links.

Reply:
It is corrected.